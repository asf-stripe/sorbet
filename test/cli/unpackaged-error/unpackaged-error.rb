# typed: true

# Doesn't matter what the contents are. This file is unpackaged and will have an error.
