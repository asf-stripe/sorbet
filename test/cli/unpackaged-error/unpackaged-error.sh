main/sorbet --silence-dev-message --stripe-packages test/cli/unpackaged-error/ 2>&1
