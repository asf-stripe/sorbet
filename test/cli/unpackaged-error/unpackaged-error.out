unpackaged-error.rb:1: File `test/cli/unpackaged-error/unpackaged-error.rb` does not belong to a package; add a `__package.rb` file to one of its parent directories https://srb.help/8006
     1 |# typed: true
        ^
Errors: 1
